datalad/interface/tests/test_save.py

```diff
@@ -81,7 +81,9 @@ def test_save(path):
     with open(opj(subds.path, "some_file.tst"), "w") as f:
         f.write("something")
     subds.add('.')
-    ok_clean_git(subds.path, annex=isinstance(ds.repo, AnnexRepo))
+    ok_clean_git(subds.path, annex=isinstance(subds.repo, AnnexRepo))
```

Comment:
so you think itwas a bug/typo before on testing subds being the same annex'ness as ds?